frontend/packages/pipelines-plugin/src/topology/pipelinesTopologyPlugin.ts

```diff
@@ -8,7 +8,11 @@ import { TopologyDecoratorQuadrant } from '@console/topology/src/topology-types'
 import { FLAG_OPENSHIFT_PIPELINE } from '../const';
 import { tknPipelineAndPipelineRunsWatchResources } from '../utils/pipeline-plugin-utils';
 import { getPipelineRunDecorator } from './build-decorators';
-import { getDataModelReconciler } from './index';
+
+const getDataModelReconciler = () =>
+  import(
+    './getPipelinesDataModelReconciler' /* webpackChunkName: "operators-topology-components" */
```

Comment:
bad chunk name

Reply:
This code was just moved here from `frontend/packages/pipelines-plugin/src/topology/index.ts` to avoid pulling in `@patternfly/react-topology into the main vendor bundle. Seems like it always had a bad chunk name.